index.html

```diff
@@ -411,6 +417,22 @@ <h3>
         </tr>
       </tbody>
     </table>
+    <section id="sec-priv-considerations">
+      <h2>
+        Security and Privacy Considerations
+      </h2>
+      <p class="note" title="Work in Progress">
+        This section is a work in progress as this document evolves.
+      </p>
+      <p>
+        The documents listed below outline various privacy and security considerations for Digital Credentials, both broadly and for presentation on the web. Their contents will be integrated into this document gradually.
+      </p>
+      <ul>
+        <li><a href="https://github.com/WICG/digital-credentials/blob/main/explainer.md">Digital Credentials API Explainer</a></li>
+        <li><a href="https://github.com/WICG/digital-credentials/blob/main/horizontal-reviews/security-privacy.md">TAG Security and Privacy Considerations Questionnaire (WIP)</a></li>
+        <li><a href="https://github.com/w3cping/credential-considerations">W3C Privacy Interest Group (PING) Credentials Considerations repository</a> which contains in-progress list of concerns and implications of identity-verification and credential presentation, for consideration in Internet and Web standardization</li>
+      </ul>
+    </section>
```

Comment:
As much as I like the explainer, I'm not sure it's worth listing it tbh. It's also good to include the accessibility considerations while we are here. 

```suggestion
    <section class="informative">
      <h2>
        Security Considerations
      </h2>
      <p class="issue" title="Work in Progress">
        This section is a work in progress as this document evolves.
      </p>
      <p>
        The documents listed below outline initial security considerations for Digital Credentials, both broadly and for presentation on the web. Their contents will be integrated into this document gradually.
      </p>
      <ul>
        <li><a href="https://github.com/WICG/digital-credentials/blob/main/horizontal-reviews/security-privacy.md">TAG Security and Privacy Considerations Questionnaire (WIP)</a></li>
      </ul>
    </section>
    <section class="informative">
      <h2>
        Privacy Considerations
      </h2>
      <p class="issue" title="Work in Progress">
        This section is a work in progress as this document evolves.
      </p>
      <p>
        The documents listed below outline various privacy considerations for Digital Credentials, both broadly and for presentation on the web. Their contents will be integrated into this document gradually.
      </p>
      <ul>
        <li><a href="https://github.com/WICG/digital-credentials/blob/main/horizontal-reviews/security-privacy.md">TAG Security and Privacy Considerations Questionnaire (WIP)</a></li>
        <li><a href="https://github.com/w3cping/credential-considerations">W3C Privacy Interest Group (PING) Credentials Considerations repository</a> which contains in-progress list of concerns and implications of identity-verification and credential presentation, for consideration in Internet and Web standardization</li>
      </ul>
    </section>
    <section class="informative">
      <h2>
        Accessibility Considerations
      </h2>
      <p class="issue" title="Work in Progress">
        This section is a work in progress as this document evolves.
      </p>
    </section>
```